modules/swagger-codegen/src/main/resources/go-server/main.mustache

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	sw "./{{apiPath}}"
```

Comment:
Relative imports (import strings that begin with `.`) aren't kosher. This needs to be a fully-qualified path.